src/content/components/accordion/style.mdx

```diff
@@ -37,7 +37,7 @@ There is no limit to the height of an open row, however, follow the other paddin
 | `.bx--accordion__content`      | padding-left               | 16 / 1     | $spacing-05 |
 | `.bx--accordion__item--active` | padding-bottom             | 24 / 1.5   | $spacing-06 |
 
-<ImageComponent caption="Structure and spacing measurements for accordion | px / rem" fixed="default">
+<ImageComponent caption="Structure and spacing measurements for accordion | px / rem" fixed="default" zoom="true">
```

Comment:
we should revert this now that we know its working

Reply:
Done.